src/plugins/discover/public/__mocks__/__storybook_mocks__/with_discover_services.tsx

```diff
@@ -146,19 +149,7 @@ export const services = {
       ui: { SearchBar, AggregateQuerySearchBar: SearchBar },
     } as unknown as UnifiedSearchPublicPluginStart,
   }),
-  theme: {
```

Comment:
We have a similar situation with our services mock here: https://github.com/elastic/kibana/blob/88871faf60ec5a62c4aea48911883a6d7f937777/src/plugins/discover/public/__mocks__/services.ts#L177-L180

It's not causing test failures at the moment so it's not a big issue, but it could be worth doing the thing as here to be sure for the future.

Reply:
@davismcphee Updated. Thanks for catching it!